transport/tchannel/channel_outbound.go

```diff
@@ -159,7 +159,7 @@ func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*tr
 	// Inject tracing system baggage
 	reqHeaders := tchannel.InjectOutboundSpan(call.Response(), req.Headers.Items())
 
-	if err := writeRequestHeaders(ctx, format, reqHeaders, call.Arg2Writer); err != nil {
+	if err := writeRequestHeaders(ctx, format, reqHeaders, call.Arg2Writer, o.transport.exactCaseHeader); err != nil {
```

Comment:
instead of injecting outbound span into `req.Header.Items()`, can we just do:
```

reqHeaders := req.Headers.Items()
if o.transport.exactCaseHeader {
  reqHeaders = req.Headers.ExactItems()
}
// inject span from reqHeaders
```